changelog/195.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added the `~sunkit_image.stara` module which is an implementation of Sunspot Tracking And Recognition (STARA).
+Added an example of plotting sunspots using the stara.
```

Comment:
Can you add a link to the example? You are able to use sphinx interlink to do this.

The easiest way is to find a changelog entry that already does it and change the filename.

Reply:
Sure